From 4b2a566a3e007184d3ee9ce5b399a63eebbdadc3 Mon Sep 17 00:00:00 2001
From: "Tuan M. Hoang" <tmhoang@flatglobe.org>
Date: Thu, 23 Feb 2017 07:53:02 +0000
Subject: [PATCH] main/parted: disable nls on s390x

Because we build with --disable-nls by default, we need to remove libintl.h in 
these files on s390x. This is specific for DASD devices only on s390x.
---
 libparted/labels/dasd.c  | 2 +-
 libparted/labels/fdasd.c | 2 +-
 libparted/labels/vtoc.c  | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/libparted/labels/dasd.c b/libparted/labels/dasd.c
index fa9414f..5acf844 100644
--- a/libparted/labels/dasd.c
+++ b/libparted/labels/dasd.c
@@ -40,8 +40,8 @@
 #include <parted/fdasd.h>
 #include <arch/linux.h>
 
-#include <libintl.h>
 #if ENABLE_NLS
+#  include <libintl.h>
 #  define _(String) dgettext (PACKAGE, String)
 #else
 #  define _(String) (String)
diff --git a/libparted/labels/fdasd.c b/libparted/labels/fdasd.c
index 1f87937..e6a9086 100644
--- a/libparted/labels/fdasd.c
+++ b/libparted/labels/fdasd.c
@@ -24,8 +24,8 @@
 
 #include <parted/parted.h>
 
-#include <libintl.h>
 #if ENABLE_NLS
+#  include <libintl.h>
 #  define _(String) dgettext (PACKAGE, String)
 #else
 #  define _(String) (String)
diff --git a/libparted/labels/vtoc.c b/libparted/labels/vtoc.c
index fdfa94f..64ba149 100644
--- a/libparted/labels/vtoc.c
+++ b/libparted/labels/vtoc.c
@@ -12,8 +12,8 @@
 
 #include <parted/parted.h>
 
-#include <libintl.h>
 #if ENABLE_NLS
+#  include <libintl.h>
 #  define _(String) dgettext (PACKAGE, String)
 #else
 #  define _(String) (String)
-- 
2.11.1

